nist/system/ksp-nist-si-16-memory-protection.yaml

```diff
@@ -0,0 +1,26 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
+# To learn more about KubeArmor visit: 
+# https://www.accuknox.com/kubearmor/ 
+
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorPolicy
+metadata:
+  name: ksp-nist-si-16-memory-protection
+  namespace: default # Change your namespace
+spec:
+  tags: ["nist", "nist-800", "si-16", "unauthorized-code-execution"]
+  message: "Alert! unauthorized code execution detected "
+  selector:
+    matchLabels:
+      container: ubuntu # Change your labels
+  process:
+    matchPatterns:
+    - pattern: ^/proc/\*/maps$
+    - pattern: ^/proc/\*/mem$
+    - pattern: /*/mv * /dev/null
```

Comment:
for my system `mv` is under `/usr/bin/mv` so please add one more check to include this structure as well @rohitrishim